pom.xml

```diff
@@ -68,7 +68,7 @@
     <dependency>
       <groupId>org.wildfly.core</groupId>
       <artifactId>wildfly-controller-client</artifactId>
-      <version>3.0.0.Beta11</version>
+      <version>2.2.1.Final</version>
```

Comment:
`3.0.0.Beta11` is WildFly 11.0.0.Alpha1. Swarm is based on WildFly 10.1.0.Final and WildFly Core 2.2.1.Final, and I think we should stick to that.